nullaway/src/main/java/com/uber/nullaway/LibraryModels.java

```diff
@@ -244,4 +252,58 @@ public String toString() {
           + '}';
     }
   }
+
+  /** Representation of a field as a qualified class name + a field name */
+  final class FieldRef {
```

Comment:
Can we just use AutoValue for this class, since we already rely on it?  Would get rid of a lot of boilerplate.  See, e.g.:

https://github.com/uber/NullAway/blob/master/nullaway/src/main/java/com/uber/nullaway/dataflow/DataFlow.java#L318-L318

Reply:
@msridhar Sure. Used `@AutoValue` : f15b983c1b68079923bc25477ddf7c1df515d237